anomalib/data/__init__.py

```diff
@@ -7,17 +7,18 @@
 from typing import Union
 
 from omegaconf import DictConfig, ListConfig
-from pytorch_lightning import LightningDataModule
 
-from .btech import BTech
-from .folder import Folder
+from anomalib.data.base import AnomalibDataModule
+
+from .btech import BTechDataModule
+from .folder import FolderDataModule
 from .inference import InferenceDataset
-from .mvtec import MVTec
+from .mvtec import MVTecDataModule
```

Comment:
I didn't realize this naming was chosen for the CLI. I changed it because I felt the name itself was more suitable for the dataset class, not the datamodule class. It's also more in line with torchvision naming convention, where the dataset class gets the name of the dataset. But I'll revert this change then.

Reply:
I'm not against it. I'm just wondering whether adding `DataModule` suffix would be user-friendly. If yes, we could keep it

Reply:
No I agree. For CLI usage it would be better to just use `--data=MVTec`